posts/FLT-announcement.md

```diff
@@ -0,0 +1,81 @@
+description: ''
+has_math: false
+link: ''
+slug: FLT-announcement
+tags: ''
+title: The Fermat's Last Theorem Project
+type: text
+---
+
+Kevin Buzzard discusses the project to prove Fermat's Last Theorem in Lean.
+
+<!-- TEASER_END -->
+# Introduction
+
+Fermat's Last Theorem (FLT) is the claim that some abstract equation has no solutions in positive integers.
+The result has no practical applications.
+So why did the announcement of its resolution by Andrew Wiles in 1993 make the front page of the New York Times?
+
+One aspect of FLT is that the theorem is very simple to *state* (`x^n+y^n=z^n` has no solutions in positive integers if `n>=3`), and yet it was incredibly hard to *prove* (indeed, it took the mathematical community over 350 years).
+Thus it serves as a great reminder of how profound mathematics is.
+But what really justifies the interest in the problem is that over the past few centuries a huge amount of mathematical theory has been developed and invented/discovered in an attempt to resolve the problem, and this mathematics has had applications from cryptography to physics.
+Wiles might have been motivated by FLT, but ultimately his work comprised of a breakthrough in the Langlands Program, a broad collection of conjectures which are still mostly wide open, and very much of interest in 2024.
+And historically several other breakthroughs in algebraic number theory (for example the theory of factorization in number fields, and the arithmetic of cyclotomic fields) have been at least partly motivated by the desire to shed more light on FLT.
+
+The work of Wiles, completed by Taylor--Wiles, was built upon a gigantic edifice of 20th century mathematics, and furthermore the fundamental technique which Wiles introduced (a "modularity lifting theorem") has been conceptually simplified and hugely generalised in the 30 years following the publication of the original papers.
+The area continues to be a very active one today; [Frank Calegari's 2022 ICM paper](https://arxiv.org/abs/2109.14145) is a survey of what has happened since Wiles' breakthrough.
+The fact that the area is still active is part of my motivation to formalise a proof of FLT.
+
+
+# Formalisation of mathematics
+
+Formalisation of mathematics is the art of translating paper mathematics into a computer programming language rich enough to understand the concepts of theorems and proofs.
+These programming languages, also called interactive theorem provers (ITPs), have been around for decades.
+Over the last few years however, the area seems to have captured the attention of parts of the mathematics community.
+We have now seen several examples of fashionable modern research mathematics being formalised in real time, the most recent of which (at the time of writing) is Tao et al's formalisation of his work with Gowers, Green and Manners resolving the polynomial Freiman--Ruzsa conjecture.
+This breakthrough 2023 paper was formalised in Lean in three weeks flat.
```

Comment:
Do we have a canonical link for this? Tao's blog perhaps, though I don't see a "we finished it" post.

Reply:
The [blueprint](https://teorth.github.io/pfr/blueprint/index.html)? The [dependency graph](https://teorth.github.io/pfr/blueprint/dep_graph_document.html)? The Quanta Magazine [article](https://www.quantamagazine.org/a-team-of-math-proves-a-critical-link-between-addition-and-sets-20231206/)?

Reply:
The last one sounds good to me